keyboards/planck/ez/info.json

```diff
@@ -0,0 +1,115 @@
+{
+  "keyboard_name": "Planck EZ",
+  "keyboard_folder": "planck/ez",
+  "url": "https://olkb.com/planck",
```

Comment:
We can have this go to https://ergodox-ez.com/pages/planck for now - I think we'll have a better URL for it later :)

Reply:
```suggestion
  "url": "https://ergodox-ez.com/pages/planck",
```